flags/14.0/vtgate.txt

```diff
@@ -113,59 +113,58 @@ Usage of vtgate:
       --statsd_address string                                            Address for statsd client
-      --statsd_sample_rate float                                          (default 1)
-      --stderrthreshold value                                            logs at or above this threshold go to stderr (default 1)
+      --statsd_sample_rate float                                         Sample rate for statsd metrics (default 1)
+      --stderrthreshold severity                                         logs at or above this threshold go to stderr (default 1)
       --stream_buffer_size int                                           the number of bytes sent from vtgate for each stream call. It's recommended to keep this value in sync with vttablet's query-server-config-stream-buffer-size. (default 32768)
-      --tablet_filters value                                             Specifies a comma-separated list of 'keyspace|shard_name or keyrange' values to filter the tablets to watch
+      --tablet_filters strings                                           Specifies a comma-separated list of 'keyspace|shard_name or keyrange' values to filter the tablets to watch.
       --tablet_grpc_ca string                                            the server ca to use to validate servers when connecting
       --tablet_grpc_cert string                                          the cert to use to connect
       --tablet_grpc_crl string                                           the server crl to use to validate server certificates when connecting
       --tablet_grpc_key string                                           the key to use to connect
       --tablet_grpc_server_name string                                   the server name to use to validate server certificate
-      --tablet_manager_protocol string                                   the protocol to use to talk to vttablet (default grpc)
-      --tablet_protocol string                                           how to talk to the vttablets (default grpc)
-      --tablet_refresh_interval duration                                 tablet refresh interval (default 1m0s)
-      --tablet_refresh_known_tablets                                     tablet refresh reloads the tablet address/port map from topo in case it changes (default true)
+      --tablet_protocol string                                           Protocol to use to make queryservice RPCs to vttablets. (default "grpc")
+      --tablet_refresh_interval duration                                 Tablet refresh interval. (default 1m0s)
+      --tablet_refresh_known_tablets                                     Whether to reload the tablet's address/port map from topo in case they change. (default true)
       --tablet_types_to_wait string                                      wait till connected for specified tablet types during Gateway initialization
```

Comment:
how come this is a string but allowed_tablet_types is a slice?

Reply:
I have made tablet_types_to_wait as string

Reply:
also i have added that is command separated string